src/Decorator/AbstractDecorator.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Phlib\Logger\Decorator;
+
+use Psr\Log\AbstractLogger;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @package Phlib\Logger
+ */
+abstract class AbstractDecorator extends AbstractLogger
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var mixed
+     */
+    protected $config;
```

Comment:
Could perhaps just define the `$config` parameter but don't actually do anything with it in the constructor body for the abstract?

Makes it obvious that there is a second parameter, matches what is expected in the Factory, but allows the user to name the parameter/property whatever they want in their concrete implementation.

Just has the oddity of a parameter which is defined but not used..